### PR TITLE
To Do List structure

### PR DESCRIPTION
**Pull request by kamranasif506**
The following tasks have been completed

- [ ] Used Webpacks
- [ ] Used Modules
- [ ] Used index,html file as a template 
- [ ] Loaded styling and font-awesome libraries through webpack
- [ ] Set an array of to do tasks (array of objects)
- [ ] Displayed the list of tasks in html using loop on that array.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @kamranasif506, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linter configs, kindly make sure to include the `.github` folder in your repo with the correct linter configurations

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+.checkbox-list input[type="checkbox"]:checked + label::after {
+  content: "\2713"; /* Unicode checkmark symbol */
+
+  /* display: inline-block; */
+  color: #fff;
+  font-size: 12px;
+
+  /* line-height: 16px; */
+
+  /* vertical-align: middle; */
```
Comment (Ibizugbe) [inline src/style.css]: Kindly take away every commented piece of code as they make your code base look a bit messy, giving it a less professional look.😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @kamranasif506 , 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are correctly setup and they are passing👍.
- [x] Github flow have been used👍.
- [x] Webpack have been used to setup project👍.
- [x] An array of some simple todo tasks has been created and rendered dynamically on page load👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @kamranasif506,

## Highlights!
- Clean code ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**